plugins/woocommerce/src/Internal/RestApi/Routes/V4/Settings/General/Controller.php

```
@@ -0,0 +1,451 @@
+				'schema' => array( $this, 'get_schema' ),
+			)
+		);
+	}
+
+	/**
+	 * Check permissions for reading general settings.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return bool|WP_Error
+	 */
+	public function get_item_permissions_check( $request ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)